### DISCUSSION TITLE
Option for Horizontal Cover Controls

### DISCUSSION THREAD

**Discussion by kuriburi** [Blackymas/NSPanel_HA_Blueprint#374] [General]
Hi, great work on this awesome project!
I am just getting started with a couple of panels, and was wondering if we could have an option for curtain controls to be horizontal instead of the roll/cover vertical ones?
Thanks in any case!

Meantime, FYI trying to make a donation but PayPal does not allow donations to be made from my country of residence (Japan).
Any other way I can help out?

**Comment by Blackymas** [comment]
Meantime, FYI trying to make a donation but PayPal does not allow donations to be made from my country of residence (Japan).
Any other way I can help out?

-> Really? I can't imagine that

instead of the roll/cover vertical ones? -> this can only be adjusted in the HMI editor itself.

What exactly does not fit? the slider too small or inaccurate?

**Comment by edwardtfn** [comment]
- #1721
